hoomd/mpcd/MPCDReversePerturbationFlow.h

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2009-2024 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+/*!
+ * \file MPCDReversePerturbationFlow.h
+ * \brief Declaration of Reverse perturbation flow
+ */
+
+#ifndef AZPLUGINS_MPCD_REVERSE_PERTURBATION_FLOW_H_
+#define AZPLUGINS_MPCD_REVERSE_PERTURBATION_FLOW_H_
+
+#ifdef NVCC
+#error This header cannot be compiled by nvcc
+#endif
+
+#include "hoomd/ParticleGroup.h"
+#include "hoomd/Updater.h"
+#include <pybind11/pybind11.h>
+
+namespace hoomd
+    {
+namespace mpcd
+    {
+//! Reverse perturbation flow updater
+/*!
+ * MPCDReversePerturbationFlow implements reverse pertubation flow as
+ * described in Mueller-Plathe,F. Physical Review E 59.5 (1999): 4894.
+ *
+ * A flow is induced by swapping velocities in x direction based on particle position in
+ * z-direction.
+ */
+class PYBIND11_EXPORT MPCDReversePerturbationFlow : public Updater
```

Comment:
Rename the class like this since it's in the mpcd namespace
```suggestion
class PYBIND11_EXPORT ReversePerturbationFlow : public Updater
```